### DISCUSSION TITLE
Cannot add repository

### DISCUSSION THREAD

**Discussion by hscholing** [homeassistant-apps/app-cloudflared#246] [Q&A] [answered]
Running on latest HA version and want to add your addon. Ha is supervised running. When adding I get the following error. Running on Homeassistant OS, so noway to install additional Linux software. Used the link to add repository

Logger: homeassistant.components.hassio
Source: components/hassio/websocket_api.py:124 
Integration: Home Assistant Supervisor (documentation, issues) 
First occurred: 23:34:10 (6 occurrences) 
Last logged: 23:37:46

Failed to to call /store/repositories - https://github.com/brenner-tobias/addon-cloudflared/ is not a valid add-on repository
Failed to to call /store/repositories - Cmd('git') failed due to: exit code(128) cmdline: git clone -v --recursive --depth=1 --shallow-submodules https://github.com/brenner-tobias/ /data/addons/git/47d01115 stderr: 'Cloning into '/data/addons/git/47d01115'... remote: Not Found fatal: repository 'https://github.com/brenner-tobias/' not found '
Failed to to call /store/repositories - Cmd('git') failed due to: exit code(128) cmdline: git clone -v --recursive --depth=1 --shallow-submodules https://github.com/brenner-tobias/ha-addons/tree/main/cloudflared /data/addons/git/1318e527 stderr: 'Cloning into '/data/addons/git/1318e527'... fatal: repository 'https://github.com/brenner-tobias/ha-addons/tree/main/cloudflared/' not found '
Failed to to call /store/repositories - https://github.com/brenner-tobias/addon-cloudflared is not a valid add-on repository

**Comment by brenner-tobias** [comment]
This repository is where the individual add-on development happens. In order to install it, you have to add my [add-on repository](https://github.com/brenner-tobias/ha-addons) to home assistant (see also [this section of the README.md](https://github.com/brenner-tobias/addon-cloudflared#installation) of this repository). Alternatively, you can click on the button included in the just referred README.
I hope this helps.

**Comment by hscholing** [comment, marked as answer]
I did use the link on the README.md file and got this error. I just checked again and it works now. I did not do anything else then yesterdayevening. Maybe a hiccup at the GitHub side? 

 

This repository is where the individual add-on development happens. In order to install it, you have to add my add-on repository <https://github.com/brenner-tobias/ha-addons>  to home assistant (see also this section of the README.md <https://github.com/brenner-tobias/addon-cloudflared#installation>  of this repository). Alternatively, you can click on the button included in the just referred README.
I hope this helps.

**Reply by brenner-tobias** [reply]
ok, thanks for letting me know. It is working fine for me, so it might have been some sort of hickup. What's strange is that your error shows that HA wanted to add the "addon-cloudflared", that's why I expected you manually added the wrong repo. But never mind, as long as it works now, everything is fine from my perspective.
